core/Dockerfile

```diff
@@ -1,3 +1,27 @@
+### Build train-sim dynamic lib (not used in core for now, but eventually will)
+# Will be used for testing
+FROM rust:latest AS train_sim_rust_env
```

Comment:
Don't we want to make a setup with `cargo chef` like in other Rust projects of `osrd`?